packages/react-server/core/util/RequestLocalStorage.js

```diff
@@ -1,5 +1,21 @@
 var RequestLocalStorage = require('request-local-storage');
+var Promise = require('bluebird');
 
-RequestLocalStorage.patch(require('cls-q'));
+RequestLocalStorage.patch(require('cls-bluebird'));
+
+// Unfortunately `cls-bluebird` doesn't go far enough, and the maintainer
+// doesn't seem to be receptive to patches.  There's been a PR open with a
+// similar patch since 2014.  Not going to wait on that.
```

Comment:
Thanks.  Added a link in the comment.

I'll also link here, so it shows up in the PR over there: https://github.com/TimBeyer/cls-bluebird/pull/1